# HG changeset patch
# User Rik <rik@octave.org>
# Date 1387413188 28800
#      Wed Dec 18 16:33:08 2013 -0800
# Node ID 1c22f0465c741bdbfffa372f621f18e9c896f870
# Parent  190ef1764d30a9baea0e73399d0a44ea93f31cce
doc: Add documentation about using set() to get default graphic properties.

* plot.txi: Add documentation about using set() to get default graphic
properties.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -1221,28 +1221,31 @@ current figure.
 To set graphics object properties, use the set function.  For example,
 
 @example
 set (gca (), "xlim", [-10, 10]);
 @end example
 
 @noindent
 sets the range of the x-axis for the current axes object in the current
-figure to @samp{[-10, 10]}.  Additionally, calling set with a graphics
-object index as the only argument returns a structure containing the
-default values for all the properties for the given object type.  For
-example,
+figure to @samp{[-10, 10]}.
+
+Default property values can also be queried if the @code{set} function is
+called without a value argument.  When only one argument is given (a graphic
+handle) then a structure with defaults for all properties of the given object
+type is returned.  For example,
 
 @example
 set (gca ())
 @end example
 
 @noindent
-returns a structure containing the default property values for axes
-objects.
+returns a structure containing the default property values for axes objects.
+If @code{set} is called with two arguments (a graphic handle and a property
+name) then only the defaults for the requested property are returned.
 
 @DOCSTRING(get)
 
 @DOCSTRING(set)
 
 @DOCSTRING(ancestor)
 
 @DOCSTRING(allchild)
